manifests/init.pp

```diff
@@ -289,6 +291,7 @@
   Enum['none', 'plain', 'login', 'cram-md5'] $email_smtp_authentication = $::foreman::params::email_smtp_authentication,
   Optional[String] $email_smtp_user_name = $::foreman::params::email_smtp_user_name,
   Optional[String] $email_smtp_password = $::foreman::params::email_smtp_password,
+  Integer[600] $apache_start_timeout = $::foreman::params::apache_start_timeout
```

Comment:
Is there a reason this should be at least 600? You're also missing a trailing comma.